index.bs

```diff
@@ -872,27 +872,30 @@ Advisory Board (AB)</h4>
 <h5 id=ab-role>
 Role of the Advisory Board</h5>
 
-	The <dfn export lt="Advisory Board|AB">Advisory Board</dfn> provides ongoing guidance to the Team
+	The <dfn export lt="Advisory Board|AB">Advisory Board</dfn>
+	is elected by the Membership to advance the effectiveness of W3C as a community by:
+
+	<ul>
+	<li>Providing ongoing guidance to the Team
 	on issues of strategy,
 	management,
 	legal matters,
 	process,
+	community engagement,
 	and conflict resolution.
-	The Advisory Board also serves the Members
+	<li>Serving the Members
 	by tracking issues raised between Advisory Committee meetings,
 	soliciting Member comments on such issues,
 	and proposing actions to resolve these issues.
```

Comment:
```suggestion
	and proposing actions to resolve these issues.</li>
```

Reply:
That's unnecessary. <li> is a self-closing tag.

Reply:
Closing this comment thread, as per https://github.com/w3c/process/pull/1021#issuecomment-2810177522